services/minespace-web/src/components/pages/LandingPage.js

```diff
@@ -55,13 +56,29 @@ export const LandingPage = (props) => (
           Manage applications, see inspection histories, submit reports, and more.
         </Typography.Paragraph>
         {!props.isAuthenticated && (
-          <Button type="primary" size="large" className="login">
-            <a
-              href={`${COMMON_ENV.KEYCLOAK.loginURL}${MINESPACE_ENV.BCEID_LOGIN_REDIRECT_URI}&kc_idp_hint=${COMMON_ENV.KEYCLOAK.idpHint}`}
+          <div>
+            <Button
+              type="primary"
+              size="large"
+              className="login"
+              style={{ "margin-right": "10px" }}
             >
-              Log in
-            </a>
-          </Button>
+              <a
+                href={`${COMMON_ENV.KEYCLOAK.loginURL}${MINESPACE_ENV.BCEID_LOGIN_REDIRECT_URI}&kc_idp_hint=${COMMON_ENV.KEYCLOAK.bceid_idpHint}`}
+              >
+                Log in with BCeID
+              </a>
+            </Button>
+            <AuthorizationWrapper inTesting>
+              <Button type="primary" size="large" className="login">
+                <a
+                  href={`${COMMON_ENV.KEYCLOAK.loginURL}${MINESPACE_ENV.BCEID_LOGIN_REDIRECT_URI}&kc_idp_hint=${COMMON_ENV.KEYCLOAK.vcauthn_idpHint}`}
+                >
+                  Log in with Verifiable Credentials
+                </a>
+              </Button>
+            </AuthorizationWrapper>
+          </div>
```

Comment:
Is it possible to use the default log in screen with VC as an option like core does? (but remove the IDIR option) ?
![Screen Shot 2021-12-21 at 9 06 07 AM](https://user-images.githubusercontent.com/25125247/146970182-7d065fa1-3c3d-4bf5-80b7-d6d1b1582b00.png)

Reply:
That is an option, if that is preferred, but that is KC configuration that is out of code (not in GH), and would need to be replicated. 

If this is preferred by your team, I can look into it, but the two buttons with kc_idp_hint, was easier.

Reply:
I can see if any of these options will give us what we want, but it's not 'customizable' beyond that as far as I know. 

![image](https://user-images.githubusercontent.com/5376854/146973409-1fb5ebec-fb80-456a-a8d8-2fa5215ecfc8.png)

Reply:
@Jsyro confirmed with the team, the two-button approach works for now! well done.